n2y/blocks.py

```diff
@@ -469,9 +469,21 @@ class LinkPreviewBlock(WarningBlock):
 
 
 class SyncedBlock(Block):
+    def __init__(self, client, notion_data, page, get_children=True):
+        super().__init__(client, notion_data, page, get_children)
+        self.shared = self.has_children
```

Comment:
What happens if you have an empty synced block? Will `has_children` be false in that case? If so, would we be printing a spurious warning in this situation?

Reply:
Sweet, thanks for checking! This is good to know. Maybe worth adding a comment about this for our future selves?